docker_autoscaler.tf

```diff
@@ -6,9 +6,47 @@
 resource "aws_security_group" "docker_autoscaler" {
   count = var.runner_worker.type == "docker-autoscaler" ? 1 : 0
 
-  description = "Docker autoscaler security group"
+  name_prefix = "${local.name_sg}-docker-autoscaler"
   vpc_id      = var.vpc_id
-  name        = "${local.name_sg}-docker-autoscaler"
+  description = "Docker-autoscaler security group"
+
+  dynamic "egress" {
+    for_each = var.runner_worker_docker_autoscaler_egress_rules
+    iterator = each
+
+    content {
+      # ok, there is no problem with outgoing data to the internet. It's a user setting
+      # tfsec:ignore:aws-ec2-no-public-egress-sgr
+      cidr_blocks = each.value.cidr_blocks
+      # ok, there is no problem with outgoing data to the internet. It's a user setting
+      # tfsec:ignore:aws-ec2-no-public-egress-sgr
+      ipv6_cidr_blocks = each.value.ipv6_cidr_blocks
+      prefix_list_ids  = each.value.prefix_list_ids
+      from_port        = each.value.from_port
+      protocol         = each.value.protocol
+      security_groups  = each.value.security_groups
+      self            = each.value.self
+      to_port          = each.value.to_port
+      description      = each.value.description
+    }
+  }
+
+  dynamic "ingress" {
```

Comment:
issue: can we move this to an `aws_vpc_security_group_ingress_rule` resource? See {aws_security_group](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group.html)